### DISCUSSION TITLE
New Projekt keeps using cifar10-splittnn

### DISCUSSION THREAD

**Discussion by eshatkeinensinn** [NVIDIA/NVFlare#2325] [Q&A]
Hey, for a project I'm trying to apply the Cifar10 Splitlearning example to other data. When I create a similar folder with the same structure in the vertical_federated_learning folder, it always takes the standard Cifar10-splittnn files during the experiment. But I can't find the reference to the Cifar10_splittnn even if i look at the jobs and app_common files . Maybe someone can help me. greetings Jakob

**Comment by chesterxgchen** [comment]
is this the example you are looking for: https://github.com/NVIDIA/NVFlare/tree/main/examples/advanced/vertical_federated_learning/cifar10-splitnn

**Comment by YuanTingHsieh** [comment]
@eshatkeinensinn I think you were following the instructions provided
so you have run
```
export PYTHONPATH=${PWD}/src:${PWD}/../../cifar10
```

This means that your python will try to get codes from here ("takes the standard Cifar10-splittnn files").
To avoid getting codes from here, you can do either:
1. just name your custom file / classes differently.
2. export PYTHONPATH=

Let us know if these do not work for you

**Reply by YuanTingHsieh** [reply]
@eshatkeinensinn feel free to open if this issue is not resolved.
